### ISSUE TITLE
Exception in safe_eval() when using lookup on a file containing UTF-8 encoded characters

### ISSUE THREAD

**Issue by ggrossetie** [ansible/ansible#23075]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
module `lookup`
http://docs.ansible.com/ansible/playbooks_lookups.html#intro-to-lookups-getting-file-contents

##### ANSIBLE VERSION
```
ansible --version
```
```
ansible 2.2.0.0
  config file = /.../project/ansible.cfg
  configured module search path = ['./library/']
```

##### CONFIGURATION
```
[defaults]
forks=25
filter_plugins = ./library/filters
callback_plugins = ./library/callbacks
lookup_plugins = ./library/lookup
vars_plugins = ./library/vars_plugins
roles_path = ./roles/:./deprecated_roles/
library=./library/
host_key_checking=False
vault_password_file = ./vault.key
module_lang = en_US.UTF-8
module_set_locale = True

[ssh_connection]
ssh_args = -o ControlMaster=auto -o UserKnownHostsFile=/dev/null
control_path = %(directory)s/%%h-%%r
timeout = 30
pipelining=True
```

##### OS / ENVIRONMENT

Red Hat Enterprise Linux Server release 6.6 (Santiago)

##### SUMMARY

The following `WARNING` is printed when using `lookup('file', 'role.json')` where `role.json` contains UTF-8 encoded characters:

```
[WARNING]: Exception in safe_eval() on expr: {   "password" : "xyz",   "roles" : [ "foo", "bar"],   "full_name" : "Utilisateur technique ayant les droits de lecture sur les données",   "enabled": true } 
('ascii' codec can't encode character u'\xe9' in position 166: ordinal not in range(128))
```
NOTE: The content has been updated to remove sensitive information, as a consequence, position 166 is invalid. But the issue remains the same because I have an UTF-8 encoded character `é` in the word `données` (data in French).

##### STEPS TO REPRODUCE

.play.yml
```yml
---
- hosts: localhost
  vars:
    content: "{{ lookup('file', 'role.json') }}"
  tasks:
    - debug:
        var: "{{ content }}"
```
.role.json
```json
{"password": "xyz", "roles": ["foo","bar"], "full_name": "Utilisateur technique ayant les droits de lecture sur les données", "enabled": true}
```

.inventory
```
localhost
```

```
ansible-playbook play.yml -i inventory
```
```
PLAY [localhost] ***************************************************************

TASK [setup] *******************************************************************
ok: [localhost]

TASK [debug] *******************************************************************
 [WARNING]: Exception in safe_eval() on expr: {"password": "xyz", "roles": ["foo","bar"], "full_name":
"Utilisateur technique ayant les droits de lecture sur les données", "enabled": true} ('ascii' codec
can't encode character u'\xe9' in position 120: ordinal not in range(128))

ok: [localhost] => {
    "{\"password\": \"xyz\", \"roles\": [\"foo\",\"bar\"], \"full_name\": \"Utilisateur technique ayant les droits de lecture sur les données\", \"enabled\": true}": {
        "enabled": true,
        "full_name": "Utilisateur technique ayant les droits de lecture sur les données",
        "password": "xyz",
        "roles": [
            "foo",
            "bar"
        ]
    }
}

PLAY RECAP *********************************************************************
localhost                  : ok=2    changed=0    unreachable=0    failed=0


PLAY TIMING ********************************************************************
default ----------------------------------------------------------------0:00:00

default | --------------------------------------------------------------0:00:00
```

##### EXPECTED RESULTS

No `WARNING` message.

##### ACTUAL RESULTS

A `WARNING` is printed.

**Comment by abadger** [comment]
This appears to have been fixed in Ansible-2.2.1.  I can reproduce the warning message in a checkout of ansible-2.2.0.0 but not in a checkout of ansible-2.2.1.0, ansible-2.2.2.0, or the current release candidate of ansible-2.3.0.

Please upgrade as it appears to fix your problem.  Thank you!

**Comment by ggrossetie** [comment]
Sorry, I should have test with the latest release :flushed: 
Anyway, I can confirm that the warning is gone after upgrading to Ansible 2.2.1.

Thanks for your help! :+1:
